src/commands/basemaps-github/create-pr.ts

```diff
@@ -121,7 +121,7 @@ export const basemapsCreatePullRequest = command({
         const info = await parseTargetInfo(target, args.individual);
         layer.name = info.name;
         layer.title = info.title;
-        layer[info.epsg] = target;
+        layer[info.epsg] = target.endsWith('/') ? target.slice(0, -1) : target;
```

Comment:
This seems like the sort of thing which should fail validation rather than trying to correct bad input. Or is this an instance of correct input which nevertheless has to be transformed?

Reply:
This is out of date, we are working on adding slash at the end of all the s3 paths. So this PR will close later.

Reply:
I'm confused - why keep this open if we know we'll close it later?